### DISCUSSION TITLE
Digital Signature Expiry

### DISCUSSION THREAD

**Discussion by JamesRNorfolk** [MscrmTools/XrmToolBox#1158] [Q&A] [answered]
Hi all. I've noticed that the current version of XrmToolBox (1.2022.5.56) has a digital signature attached to it, valid for one year, which expires at the end of August 2022. Does anybody know whether a new signature will be issued after this date?

My thought is that a digital certificate is issued on a yearly basis, but I didn't just want to presume, so if anybody in the community knows whether that is true or not, that would be very helpful.

Kind regards
James

**Comment by MscrmTools** [comment, marked as answer]
Don't worry, the digital signature is still valid after august. And for new versions after that, a new digital signature will be attach

**Reply by JamesRNorfolk** [reply]
Thanks very much, that's really helpful. I've only been using the application since around November last year so wanted to make sure!
